### DISCUSSION TITLE
How to install WiX v4 & v5 without access to NuGet?

### DISCUSSION THREAD

**Discussion by rainbow180** [wixtoolset/issues#8190] [Questions]
Hi, I'm hoping there is a quick answer to this.
We’re current Wix v3 users on a large product (at a large corp.) and are hoping to take advantage of all the new features/support in WiX v5. However, unlike WiX v3 it seems that v5 is only available to download and install from NuGet.org. Unfortunately, we are blocked from using NuGet, so we need a way to install everything without calling out to the external NuGet.org repository. I'm currently banging my head into the preverbal brick wall trying to get this block lifted ☹

Please tell me there is an alternative work-around!

Thanks Antony

**Comment by Thylon1** [comment]
Hi Antony,
if you do not want to build from source, your only option is probably to download the packages manually and set up your own nuget feed:
https://learn.microsoft.com/en-us/nuget/hosting-packages/overview

**Comment by barnson** [comment]
You can get all the packages from the GitHub release.

**Comment by chrpai** [comment]
I've done it like this for customers:

<img width="678" alt="image" src="https://github.com/wixtoolset/issues/assets/43178814/a04195fe-e038-4110-b1b9-0bcb80d79c3e">

Nuget.config in the .sln folder.

```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <packageSources>
    <clear />
    <add key="local" value=".\Nuget" />
  </packageSources>
</configuration>

```

**Comment by rainbow180** [comment]
Thanks for all the answers. I'm slowly starting to understand how to host the packages locally within the project.
This hasn't made the Heatwave migration tool from v3 to v4(5) very easy but I think I've got our bundle project and all the associated sub msi project migrated now.
